### ISSUE TITLE
BungeeCord saying I'm already connected.

### ISSUE THREAD

**Issue by ProMiner10000** [SpigotMC/BungeeCord#1738]
16:30:16 [INFO] Using standard Java JCE cipher. To enable the OpenSSL based native cipher, please make sure you are using 64 bit Ubuntu or Debian with libssl installed.
16:30:16 [INFO] Using standard Java compressor. To enable zero copy compression, run on 64 bit Linux
16:30:16 [INFO] Enabled BungeeCord version git:BungeeCord-Bootstrap:1.8-SNAPSHOT:848cad2:1116
16:30:17 [INFO] Discovered module: ModuleSpec(name=cmd_alert, file=modules\cmd_alert.jar, provider=JenkinsModuleSource())
16:30:17 [INFO] Discovered module: ModuleSpec(name=cmd_find, file=modules\cmd_find.jar, provider=JenkinsModuleSource())
16:30:17 [INFO] Discovered module: ModuleSpec(name=cmd_list, file=modules\cmd_list.jar, provider=JenkinsModuleSource())
16:30:17 [INFO] Discovered module: ModuleSpec(name=cmd_send, file=modules\cmd_send.jar, provider=JenkinsModuleSource())
16:30:17 [INFO] Discovered module: ModuleSpec(name=cmd_server, file=modules\cmd_server.jar, provider=JenkinsModuleSource())
16:30:17 [INFO] Discovered module: ModuleSpec(name=reconnect_yaml, file=modules\reconnect_yaml.jar, provider=JenkinsModuleSource())
16:30:17 [INFO] Loaded plugin reconnect_yaml version git:reconnect_yaml:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Loaded plugin cmd_find version git:cmd_find:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Loaded plugin cmd_server version git:cmd_server:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Loaded plugin cmd_alert version git:cmd_alert:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Loaded plugin cmd_send version git:cmd_send:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Loaded plugin cmd_list version git:cmd_list:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Enabled plugin reconnect_yaml version git:reconnect_yaml:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Enabled plugin cmd_find version git:cmd_find:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Enabled plugin cmd_server version git:cmd_server:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Enabled plugin cmd_alert version git:cmd_alert:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Enabled plugin cmd_send version git:cmd_send:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Enabled plugin cmd_list version git:cmd_list:1.8-SNAPSHOT:848cad2:1116 by SpigotMC
16:30:17 [INFO] Listening on /0.0.0.0:25565
16:30:20 [SEVERE] Netty is not using direct IO buffers.
16:30:20 [INFO] [ProMiner10000] <-> InitialHandler has connected
16:30:22 [INFO] [ProMiner10000] <-> ServerConnector [lobby] has connected
16:30:22 [INFO] [ProMiner10000] <-> InitialHandler has connected
16:30:22 [SEVERE] [ProMiner10000] <-> ServerConnector [lobby] - encountered exception
java.lang.RuntimeException: Server is online mode!
at net.md_5.bungee.ServerConnector.handle(ServerConnector.java:261)
at net.md_5.bungee.protocol.packet.EncryptionRequest.handle(EncryptionRequest.java:56)
at net.md_5.bungee.netty.HandlerBoss.channelRead(HandlerBoss.java:76)
at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:318)
at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:304)
at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:318)
at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:304)
at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:276)
at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:263)
at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:318)
at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:304)
at io.netty.handler.timeout.ReadTimeoutHandler.channelRead(ReadTimeoutHandler.java:152)
at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:318)
at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:304)
at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:846)
at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:511)
at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:468)
at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:382)
at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:354)
at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:112)
at java.lang.Thread.run(Unknown Source)
16:30:22 [INFO] [ProMiner10000] disconnected with: Exception Connecting:RuntimeException : Server is online mode! @ net.md_5.bungee.ServerConnector:261
16:30:22 [INFO] [ProMiner10000] <-> ServerConnector [lobby] has disconnected
16:30:22 [INFO] [ProMiner10000] -> UpstreamBridge has disconnected

That's the error I get. ^^
http://pastebin.com/gCLp3fNH - BungeeCord Configuration
http://pastebin.com/HmhQzcRs - Server properties for Spawn
http://pastebin.com/epDT00QP - Server properties for SimplePVP
http://pastebin.com/qF8vPgSk - Server properties for Factions

Basically all I want is when they log in, they go to the server called Spawn which contains the Spawn and from there they can use a compass to teleport to SimplePVP and Factions. Also I have enabled IP-forwarding in the bungeecord config

**Comment by md-5** [comment]
#1737
